lib/services/elasticsearch.js

```diff
@@ -1231,13 +1244,33 @@ class ElasticSearch extends Service {
       });
   }
 
+  _mergeDefaultMapping (index, oldMapping, newMapping) {
+    const collectionMapping = {};
```

Comment:
As we discuss, it's complicated because the old, new and default mapping does not have the same priority for all properties (eg: `oldMapping.properties._kuzzle_info`)
And it's better to stay explicit for this part I think